src/gameinfo/gameinfo-disc-id-title.vala

```
@@ -25,4 +25,8 @@ public class Games.GameinfoDiscIdTitle : Object, Title {
 
 		return dgettext (domain, title);
 	}
+
+	public string get_raw_title () {
```

This one should throw an error.